libc-test/build.rs

```diff
@@ -2504,9 +2454,6 @@ fn test_freebsd(target: &str) {
 
             t if t.ends_with("_t") => t.to_string(),
 
-            // sigval is a struct in Rust, but a union in C:
-            "sigval" => "union sigval".to_string(),
-
```

Comment:
This change seems to make freebsd fail, but I'm not sure why or what would be the correct thing to do

Reply:
The FreeBSD jobs are unfortunately flakey with `ctest`; I've tried but I can't fix the "spooky action at a distance" response to changes that seem to happen on those.

If you can get the rests of the tests passing (looks like s390x and Solaris are failing) we can hand-review the FreeBSD changes or figure something else out there.

Reply:
The Solaris tests seems to fail for similar reasons than FreeBSD, or at least similar symptoms.
I'll investigate s390x and emscripten